src/resources/Correction/GeneralPartnershipResource.ts

```diff
@@ -0,0 +1,24 @@
+import { CorrectionTypes } from '@/enums/'
+import { CorpTypeCd } from '@bcrs-shared-components/corp-type-module/'
+
+export const GeneralPartnershipResource: any = {
+  addressLabel: 'Business Office',
+  entityType: CorpTypeCd.PARTNERSHIP,
+  entityReference: 'Company',
+  contactLabel: 'Business Office',
+  title: 'General Partnership Statement',
+  description: 'See designs.',
+  changeData: {
+    nameChangeOptions: [
+      CorrectionTypes.CORRECT_NEW_NR,
+      CorrectionTypes.CORRECT_NAME
+    ],
+    orgPersonInfo: {
+      orgPersonLabel: 'Partners',
+      orgTypesLabel: 'Corporation or Firm'
+    }
+  },
+  certifyClause: 'Note: It is an offence to make or assist in making a false or misleading statement in a record ' +
+    'filed under section 90.4 of the Partnership Act. A person who commits this offence is subject to a maximum ' +
+    'fine of $5,000.'
+}
```

Comment:
Some of these properties need to be updated according to the designs (future).

Same for SoleProprietorshipResource.ts below.